src/stdafx.h

```diff
@@ -81,29 +82,62 @@
 
 #include "zip.h"
 
+namespace fs = std::tr2::sys;
 #else
+#include "boost/filesystem.hpp"
+#include <stdarg.h>
```

Comment:
This one is actually a standard header, so it should go into the unconditional block at the beginning of the file. Also, can you make it `<cstdarg>`, for consistency with the rest?